temporalcli/commandsmd/commands.md

```diff
@@ -635,6 +635,29 @@ temporal workflow execute
 Includes options set for [workflow start](#options-set-for-workflow-start).
 Includes options set for [payload input](#options-set-for-payload-input).
 
+### temporal workflow history: Interact directly with a workflow's event history.
+
+Workflow history commands use this syntax: `temporal workflow history COMMAND [ARGS]`.
+
+### temporal workflow history convert: Converts an event history between formats.
+
+```
+temporal workflow history convert \
+                --input-format json \
+		--input-file original.json \
+                --output-format json \
+		--output-file reserialized.json
+```
+
+Use the options listed below to change the command's behavior.
+
+#### Options
+
+* `--source-format` (string-enum) - Format of the input file.  Options: json, proto, pb, prototext, pbtxt.  Default: json.
```

Comment:
I think the only use case is conversion from old to new, not for different proto formats. I know it can be seen as a "why not", but there's really only one Temporal history format, it just happens to be proto JSON.

I don't think this command really even needs to exist, but if it does, I wonder if we should call it `fix-history-format` or something.